### PR TITLE
Fix profile navigation

### PR DESCRIPTION
**Pull request by huwshimi**
Fix the section used to navigate to and from the profile.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- deploy the GUI
- open the profile
- click Manage on a model
- the model should load and the profile should close

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5102/
Test PASSed.

File: jujugui/static/gui/src/app/components/env-switcher/env-switcher.js [hunk 1]
```diff
@@ -211,7 +211,7 @@ YUI.add('env-switcher', function() {
     */
     showUserProfile: function() {
       this.props.changeState({
-        sectionC: {
+        sectionB: {
           component: 'profile',
```
Comment (bac) [inline jujugui/static/gui/src/app/components/env-switcher/env-switcher.js]: OK, this make no sense to me, but there you have it.

Comment (huwshimi) [inline jujugui/static/gui/src/app/components/env-switcher/env-switcher.js]: I changed the section the profile is displayed in, but forgot to update these changeState calls to use the new section.

**Comment by bac** [comment]
To QA I launched via GI Joe, went to the admin profile page, and clicked 'Manage' and went to the gui.  A-OK.

👍

**Comment by huwshimi** [comment]
Thanks @bac and @hatched :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**